database/ffldb/transaction_test.go

```diff
@@ -8,87 +8,93 @@ import (
 	"testing"
 )
 
-func TestKeyValueTransactionCommit(t *testing.T) {
-	// Open a test db
-	path, err := ioutil.TempDir("", "TestKeyValueTransactionCommit")
+func prepareDatabaseForTest(t *testing.T, testName string) (db database.Database, teardownFunc func()) {
```

Comment:
I think there might me a more appropriate place for this function rather transaction_test.go.
Even if file of it's own.
Even if this is currently used only in transaction_test.